mpas_analysis/ocean/time_series_ocean_regions.py

```diff
@@ -433,20 +433,22 @@ def run_task(self):  # {{{
                     units = timeSeries.units
                     description = timeSeries.long_name
 
-                    if 'nVertLevels' in timeSeries.dims:
+                    is3d = 'nVertLevels' in timeSeries.dims
```

Comment:
ooh, I didn't know one could do that.. Basically the variable becomes `True` or `False` depending on whether the rhs exists or not?! nice.

Reply:
Yep, anything you can put in an `if` is just something that evaluates to `True` or `False` so it could be stored in a variable instead.  Another example: `valid = input is not None`.

Reply:
nice.